Dockerfile

```diff
@@ -1,8 +1,11 @@
-FROM node:6
+FROM node:boron
 
 RUN apt-get update \
     && apt-get install -y \
         git \
+        # Next 2 needed for yarn
+        apt-transport-https \
```

Comment:
Michael actually wrote this section on my machine when we were getting me updated to work with docker. These steps are necessary, because the packages are not included in the default node container to add apt repo with https. We could install yarn via curl piped through bash, but it's probably less secure.